packages/sanity/src/core/studio/colorScheme.tsx

```diff
@@ -180,6 +182,12 @@ export function useColorSchemeValue(): ThemeColorSchemeKey {
  * @internal
  */
 export function useColorScheme() {
+  useEffect(() => {
+    console.warn(
+      'useColorScheme() is deprecated, use useColorSchemeValue() or useColorSchemeSetValue() instead',
+    )
+  }, [])
```

Comment:
The `@deprecated` TSDoc haven't been sufficient to discourage using this hook, which is why it makes sense to `console.warn` in `useEffect` to push for possible userland to refactor away 🙌

Reply:
Yeah, I'm in favor of only logging once when it's not actionable. I'm this case the migration is very easy to do and I'd like to encourage it as much as possible 😬